### ISSUE TITLE
Crash after a moment as inactive window.

### ISSUE THREAD

**Issue by DrawToonZ** [opentoonz/opentoonz#4184]
**Describe the bug**
I leave Opentoonz as an inactive window, for example while I consult a YouTube reference video. If after a brief moment I press the stylus in the Opentoonz interface it closes automatically. It is systematic, and I can reproduce it as many times as I want. Also, it crashes if I use the stylus while it is launching.

**To Reproduce**
List steps to reproduce the behavior.
1. Open Opentoonz, and draw something.
2. Activate other Window, for example Youtube, and wait a moment.
3. Activate Opetoonz directly with stylus.

**Expected behavior**
Don't crash.

**Video**
There is a video on #3941 that I recorded months ago **with first 1.5 release** that shows exactly the same problem that occurs with **nightly builds**.

**Device Information:**
 - OS: Microsoft Windows 10 Home.
 - OpenToonz Version: Nightly (2021-12-10)]
 - GPU: NVIDIA GeForce GTX 1050
 - GPU Driver Version: 399.01
 - DDR4 16Gb
 - HUION KAMVAS GT-191

-------
Thanks a lot.

**Comment by konero** [comment]
> GPU Driver Version: 399.01

That driver is over three years old, you should update this (whether it helps or not I don't know). I'd also recommended updating tablet drivers if they're also out of date.

**Comment by DrawToonZ** [comment]
Thank you for your help, **konero**. The tablet driver was updated, and GPU driver is now updated. Unfortunately, Opentoonz is still crashing when I repeat the same steps. I can't guess why. It's a pity.

**Comment by DarrenTAnims** [comment]
I have exactly the same issue and it's repeatable 100% of the time. I added details of my experience with this issue in another bug report (I can't find it atm) and a couple of the developers (including @shun-iwasawa) looked into it and couldn't repeat it, so unless your report helps, it's likely to not be fixed. All I can say is, try to remember to use the mouse to give OpenToonz focus.

**Comment by DrawToonZ** [comment]
Thanks for your reply, **DarrenTAnims**. How sad. I understand that they cannot fix an issue if they cannot reproduce and detect. It did not happen with version 1.4 (others did, of course), nor with Tahoma2D. I can't provide more information, I think. Hopefully it doesn't happen with version 1.6.

**Comment by artisteacher** [comment]
I have run into this on occasion on macOS - including with Tahoma2D but it's not repeatable.

**Comment by DrawToonZ** [comment]
Hello, @artisteacher. Thanks for the info. Even working in Windows, it seems that this does not happen to most. In my case, as **DarrenTAnims** says, it is repeatable as many times as I want. It's a shame not knowing what's causing it.

**Comment by RodneyBaker** [comment]
Others have mentioned this but it is well worth repeating...
Double/triple check your drivers for whatever pen you are using.
I know Wacom recently had an updated driver go out.

In a few cases drivers have to be rolled back to previous release but that is rare.
It is usually best to maintain the latest driver.

If you are running multiple instances of Opentoonz try to watch to see if that relates to this crashing issue.
The same goes for running Opentoonz and a fork at the same time as they may share some code/code space.
As such as trigger for one event could (theoretically) trigger the other program.

**Comment by DrawToonZ** [comment]
Hi @RodneyBaker. Thanks for the reminder.
The driver of the tablet is updated, and I also updated the one for the graphics card, after **konero**'s advice. I just checked them again, right now, after your message.
Unfortunately, I just ran several tests with only one OpenToonz instance open, with no other programs open, and the result is the same as can be seen in this video ( #3941 ).
It only happens when I use the stylus, not with the mouse, so I suppose it is related to the input of the tablet, but it does not happen to me with any other program.
If I perceive something that could be a clue, I will report.

Thanks a lot.
